### ISSUE TITLE
FAQ of the details of the Google Drive permissions scopes requested for app.diagrams.net

### ISSUE THREAD

**Issue by davidjgraph** [jgraph/security-privacy-legal#10]

**Comment by oly-sean-mullins** [comment]
When a user connects their Google Drive to app.diagrams.net, is any PII transmitted and stored by app.diagrams.net or any subprocessors that you work with?  We are trying to confirm that there is no personal student information being shared in order to be in compliance with our state student data privacy laws.  Thank you.

**Comment by davidjgraph** [comment]
@oly-sean-mullins No. PII is never transmitted or stored any under condition by us. I don't know the answer of that for Google, though. We don't transmit anything to Google.
